src/WinUIExSample/MainWindow.xaml

```diff
@@ -2,16 +2,18 @@
     x:Class="WinUIExSample.MainWindow"
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    xmlns:local="using:WinUIExSample"
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="using:WinUIExSample"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     xmlns:winex="using:WinUIEx"
-    TaskBarIcon="Images/WindowIcon.ico"
     Title="WinUI EX Test App"
-    Width="1024" Height="768"
-    MinWidth="500" MinHeight="250"
+    Width="1024"
+    Height="768"
+    MinWidth="500"
+    MinHeight="250"
+    TaskBarIcon="Images/WindowIcon.ico"
     mc:Ignorable="d">
-    
+
```

Comment:
These changes seem unrelated?

Reply:
My bad, I forgot I was using the XAML formatter extension.  This is my first time contributing to a open source project.

Reply:
No worries. You're doing awesome! Keep it up.
As a future tip, when you go to submit a new PR, you should see all the changes below. Always good to scroll over it and make sure only the changes you really need are in there before you hit that submit button. Formatting and cleanup of unrelated code just makes it a lot harder on the maintainers.